src/js/main.js

```diff
@@ -3,21 +3,32 @@ import { loadHeaderFooter } from "./utils";
 import { testimonial } from "./testimonial";
 import RecipeListingSide from "./RecipeListingSide.mjs"
 import ExternalServices from "./ExternalServices.mjs";
+import initRouter from './router';
 
-loadHeaderFooter();
+console.log("INIT");
 
 // Four recipes on the right side on main
-const dataSource = new ExternalServices();
-const element = document.querySelector(".recipeContainer");
-const listing = new RecipeListingSide(dataSource, element);
+let title =  window.location.pathname
+console.log(title.length)
+if (title == "index.html" || title.length < 2){
+// const dataSource = new ExternalServices();
+// const element = document.querySelector(".recipeContainer");
+// const listing = new RecipeListingSide(dataSource, element);
```

Comment:
@JosueBCe Here is an important piece. I had to comment all of  these out because they are trying to access HTML elements that aren't available yet. This stuff will have to happen after the router has done it's job. This is some of the messiness caused by using HTML + Javascript here. But it's not terribly difficult to address. You just need to have some way to do stuff after the route changes. I'll let you be creative there.